### ISSUE TITLE
HoldFixed causes value(<fixed_variable>) to return 0.0

### ISSUE THREAD

**Issue by MartinBonde** [GAMS-dev/gams.jl#13]
A minor issue, but I thought I would report it in case it can be easily fixed or others stumble upon it.

To reproduce:
```
using JuMP
using GAMS

model = Model(GAMS.Optimizer)
set_optimizer_attribute(model, "HoldFixed", 1)
set_optimizer_attribute(model, GAMS.ModelType(), "CNS")

@variables model begin
    x
    y
end

@constraints model begin
    y == x
end

fix(x, 1.0)
optimize!(model)

@assert value(x) == value(y) "$(value(x)) == $(value(y))"
```
`AssertionError: 0.0 == 1.0`

```
set_optimizer_attribute(model, "HoldFixed", 0)
optimize!(model)
"$(value(x)) == $(value(y))"
```
`"1.0 == 1.0"`

**Comment by MartinBonde** [comment]
Turns out this is not directly due to HoldFixed, but rather it is the case for any variable not actually included in the model:

```
using JuMP, GAMS
model = Model(GAMS.Optimizer)

@variables model begin
    x
    y
end
fix(x, 1)

@constraints model begin
    y == 1
end

optimize!(model)
@assert value(x) == value(y) "$(value(x)) == $(value(y))"
```
`AssertionError: 0.0 == 1.0`

**Comment by renkekuhlmann** [comment]
I can confirm the bug. I'll look into it. Thanks very much for reporting!

**Comment by renkekuhlmann** [comment]
Fixed with v0.3.2, will be available in a moment.
